x-pack/plugins/ml/public/application/explorer/_explorer.scss

```diff
@@ -328,3 +170,161 @@
     }
   }
 }
+
+.ml-swimlanes {
```

Comment:
Might be an opportunity to rename this to `mlSwimlanes`. We could also add a comment that this bunch of rather brittle CSS was done to allow us to style the `DragSelect` library based classes we're using (https://github.com/ThibaultJanBeyer/DragSelect)

Reply:
I just moved these styles to the upper scope. The changes you propose might deserve a dedicated PR 🤔